### PR TITLE
Create logs.md

### PR DESCRIPTION
**Pull request by beckettsean**
@rkuchan @rossmcdonald @gunnaraasen @kostasb 

Trying to make Enterprise logging clearer. I may have missed something about adding a new page, and I'm not sure my tech is correct or comprehensible.

### CODE REVIEW

File: content/enterprise/v1.0/administration/logs.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Data Nodes
+
+For data nodes the <node-type> is `influxdb`.
+The default log command is `sudo journalctl -u influxdb.service`
+The service configuration file is `/etc/default/influxdb`.
+
+#### Enterprise Web
+
+For data nodes the <node-type> is `influxdb-enterprise`.
```
Comment (rossmcdonald) [inline content/enterprise/v1.0/administration/logs.md]: The node type for enterprise is `influx-enterprise`, not `influxdb-enterprise`. The same applies to the service configuration file, which is located at `/etc/default/influx-enterprise`.

File: content/enterprise/v1.0/administration/logs.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Data Nodes
+
+For data nodes the <node-type> is `influxdb`.
+The default log file is `/var/log/influxdb/influxdb.log`
+The service configuration file is `/etc/default/influxdb`.
+
+#### Enterprise Web
+
+For Enterprise Web nodes the <node-type> is `influxdb-enterprise`.
```
Comment (rossmcdonald) [inline content/enterprise/v1.0/administration/logs.md]: This should also be changed to `influx-enterprise`.

**Comment by rossmcdonald** [comment]
Two comments on changing `influxdb-enterprise` to `influx-enterprise`, but +1 other than that.

**Comment by beckettsean** [comment]
Thanks for the notes, @rossmcdonald! I had it right originally and then got copy/paste happy during a reformat. All fixed now, I think.

@rkuchan this look good to you?

File: content/enterprise/v1.0/administration/logs.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+### sysvinit
+
+If InfluxDB was installed using a pre-built package, and then launched
+as a service, `stderr` is redirected to
+`/var/log/influxdb/<node-type>.log`, and all log data will be written to
+that file.  You can override this location by setting the variable
+`STDERR` in the file `/etc/default/<node-type>`.
+
+For example, if on a Data node `/etc/default/influxdb` contains:
```
Comment (rkuchan) [inline content/enterprise/v1.0/administration/logs.md]: super nit: Data -> data.

**Comment by rkuchan** [comment]
Just one comment. Otherwise everything else looks good!

**Pull request merged**